airflow/providers/trino/hooks/trino.py

```diff
@@ -82,12 +111,15 @@ def get_conn(self) -> Connection:
                 delegate=_boolify(extra.get('kerberos__delegate', False)),
                 ca_bundle=extra.get('kerberos__ca_bundle'),
             )
+
+        http_headers = {"X-Trino-Client-Info": generate_trino_client_info()}
```

Comment:
The `X-Trino` header prefix has been configurable since Trino was rebranded from PrestoSQL:

https://trino.io/docs/current/admin/properties-general.html?highlight=headers#protocol-v1-alternate-header-name

Do you think that its worth supporting the same kind of override here? If a user has overriden the default header prefix then this hardcoded value of `X-Trino-Client-Info` won't get properly picked up or displayed in the Trino UI.

Reply:
I'm OK with not supporting it.
Users of older versions can overide the function and have whatever custom behavior they need to support older versions.